test/test_selectors.py

```diff
@@ -16,7 +16,10 @@
 except ImportError:
     from time import time as get_time
 
-import resource
+try:  # Windows doesn't have the resource module.
+    import resource
```

Comment:
The `try...except` exists in master but uses Python 2.6 as the justification for its presence. I removed that block a while back because v2 doesn't support 2.6.